wandb/sdk/artifacts/storage_handlers/azure_handler.py

```diff
@@ -136,10 +136,7 @@ def store_path(
             name_starts_with=f"{blob_name}/"
         ):
             if len(entries) >= max_objects:
-                raise ValueError(
-                    f"Exceeded {max_objects} objects tracked, pass max_objects to "
-                    f"add_reference"
-                )
+                break
```

Comment:
do you you want to print a warning to the user, perhaps?

Reply:
added this!